exchangeRate.go

```diff
@@ -37,11 +37,25 @@ func NewExchangeRate() *ExchangeRate {
 // Parse provides no guarantee about all fields being filled in. Callers should make a Validate() call to confirm
 // successful parsing and data validity.
 func (eRate *ExchangeRate) Parse(record string) error {
-	if utf8.RuneCountInString(record) != 18 {
-		return NewTagWrongLengthErr(18, len(record))
+	if utf8.RuneCountInString(record) < 6 {
+		return NewTagMinLengthErr(6, len(record))
 	}
+
 	eRate.tag = record[:6]
-	eRate.ExchangeRate = eRate.parseStringField(record[6:18])
+
+	var err error
+	length := 6
+	read := 0
+
+	if eRate.ExchangeRate, read, err = eRate.parseVariableStringField(record[length:], 12); err != nil {
```

Comment:
Do we want to always overwrite the data? 

```go
exchangeRate, read, err := eRate.parseVariableStringField(record[length:], 12)
if err != nil {
	return fieldError("ExchangeRate", err)
}
eRate.ExchangeRate = exchangeRate
```

Reply:
fixed